bridges/PhoronixBridge.php

```diff
@@ -6,17 +6,59 @@ class PhoronixBridge extends FeedExpander {
 	const URI = 'https://www.phoronix.com';
 	const CACHE_TIMEOUT = 3600;
 	const DESCRIPTION = 'RSS feed for Linux news website Phoronix';
+	const PARAMETERS = array(array(
+		'n' => array(
+			'name' => 'Limit',
+			'type' => 'number',
+			'required' => false,
+			'title' => 'Maximum number of items to return',
+			'defaultValue' => 10
+		),
+		'svgAsImg' => array(
+			'name' => 'SVG in "image" tag',
+			'type' => 'checkbox',
+			'title' => 'Some benchmarks are exported as SVG with "object" tag,
+but some RSS readers don\'t support this. "img" tag are supported by most browsers',
+			'defaultValue' => false
+		),
+	));
 
 	public function collectData(){
-		$this->collectExpandableDatas('https://www.phoronix.com/rss.php', 15);
+		$this->collectExpandableDatas('https://www.phoronix.com/rss.php', $this->getInput('n'));
 	}
 
 	protected function parseItem($newsItem){
 		$item = parent::parseItem($newsItem);
 		// $articlePage gets the entire page's contents
 		$articlePage = getSimpleHTMLDOM($newsItem->link);
```

Comment:
```suggestion
		$articlePage = getSimpleHTMLDOM($newsItem->link);
		$articlePage = defaultLinkTo($articlePage, $this->getURI());
```

Reply:
this should fix the issue that the posts have a category image which's link is relative:

![image](https://user-images.githubusercontent.com/16492843/160103361-0fc13e72-d3bf-4fd2-85bf-6167389fd287.png)

Reply:
[defaultLinkTo](https://rss-bridge.github.io/rss-bridge/Helper_functions/index.html#defaultlinkto) description

Reply:
Thanks for the review and the suggestion. I didn't notice this because TT-RSS was doing the job itself…